ChildProject/pipelines/metricsFunctions.py

```diff
@@ -51,6 +51,14 @@ def new_func(annotations: pd.DataFrame, duration: int, **kwargs):
     return decorator
             
 
+@metricFunction({"speaker"},{"speaker_type"}) 
+def voc_speaker(annotations: pd.DataFrame, duration: int, **kwargs):
+    """number of vocalizations for a given speaker type
+    
+    Required keyword arguments:
+        - speaker : speaker_type to use
+    """
+    return annotations[annotations["speaker_type"]== kwargs["speaker"]].shape[0]
 
```

Comment:
I added in the available metrics the voc_speaker metrics because I figured it made sense to have it available when extracting from segments (as when doing segment extraction, it is unlikely you want rates instead of actual count numbers